ui/insight/src/global.d.ts

```diff
@@ -0,0 +1,71 @@
+// Extend the global lichess object with a function defined in
+// public/javascripts/insight-refresh.js
+interface Lichess {
+  refreshInsightForm(): void;
+}
+
+/* Type definitions for multiple-select.js */
+
+// multiple-select.js uses jquery, but typescript thinks $ is Cash, not jquery.
+// So as a hack, we extend the Cash interface knowing that at runtime, it will
+// actually be a jquery object.
```

Comment:
I wasn't sure if there was a way to import Lichess types without making cash conflict with jquery. Cash functions and types are close enough to jquery that no casting is required, so for now I'm using cash types on jquery values. It works, but it feels bad. Maybe there's a better way?

Reply:
This has nothing to do with jQuery. Pretty sure we never use jQuery. This is just a plugin that is only loaded on this page. Although the plugin was originally written for jQuery, the version we use is adjusted for cash.

The type definitions still make sense though but you should change the comment.

Reply:
We include a stripped version of jQuery 3.1.1: https://github.com/ornicar/lila/blob/85f0625df4dc3607c05f6356b31542741899868a/app/views/insight.scala#L27

I'll try taking it out and seeing what happens

Reply:
Huh, interesting. It's possible I might be wrong then and this plugin hasn't been adjusted. Probably would still be nice to get rid of this but if it doesn't immediately work this can definitely be left for later.

Reply:
Yeah, once jQuery is removed, clicking on filter checkboxes triggers a rerender but doesn't toggle the checkbox.

Reply:
I rewrote all lichess jQuery plugin for Cash, except for this one, because it was too much effort. That's why jQuery is still required on the insights page.